azure-pipelines/template-debugagent.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+- script: |
+    set -x
+    sudo df -h
+    sudo ls -al /
+    sudo ls -al /nfs
+    sudo ls -al /data
+    sudo getfacl /mnt /data
+    docker info
+  condition: always()
+  displayName: Debug agent issues
```

Comment:
Can we change to a better display name? For example, show agent info.
Can we merge two templates into one?

Reply:
This step is designed to run after build job.